### PR TITLE
support .mjs as es modules and js mime type

### PR DESCRIPTION
**Pull request by Jxck**
node.js decided to use `.mjs` for es modules
and modules are same mime type with `.js`
https://github.com/nodejs/node-eps/blob/master/002-es-modules.md#321-mime-of-mjs-files

### CODE REVIEW

File: lib/handler/mimemap/defaults.c.h [hunk 1]
```diff
@@ -41,6 +41,7 @@ MIMEMAP("jpeg", "image/jpeg")
 MIMEMAP("jpg", "image/jpeg")
 MIMEMAP("jxr", "image/jxr")
 MIMEMAP("js", "application/javascript")
+MIMEMAP("mjs", "applicaftion/javascript")
```
Comment (deweerdt) [inline lib/handler/mimemap/defaults.c.h]: Hello @jxck , thank you for opening the PR. It looks like there's a typo here in `application`

Comment (Jxck) [inline lib/handler/mimemap/defaults.c.h]: oops

**Comment by deweerdt** [comment]
👍  873fc3c

**Comment by Jxck** [comment]
FYI: https://tools.ietf.org/html/draft-bfarias-javascript-mjs-01

**Comment by kazuho** [comment]
@Jxck @deweerdt Thank you!

**Pull request merged**